packages/cyberstorm/src/components/componentStyles/TextInput.module.css

```diff
@@ -0,0 +1,60 @@
+/* Common textInput styles */
+.root {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 0;
+  gap: var(--gap-m);
+  align-items: center;
+
+  padding: var(--padding-xs) var(--padding-m);
+  border: var(--size-6xs) solid var(--color-default-purple--3);
```

Comment:
There seems to exists `--border-width-m` that could be used instead of `--size-6xs`. (There are other occurances of this in the same file.)